pages/css/bootstrap-responsive.css

```diff
@@ -24,7 +24,7 @@
 }
 
 .hide-text {
-  font: 0/0 a;
+  font: 0/0 a, sans-serif;
```

Comment:
This might break the intention here: https://stackoverflow.com/a/20641095/2546338
I suggest replacing this by

```
font-size: 0;
line-height: 0;
```

There are other places in the PR with this, I won't mention them individually.

Reply:
So result will be 

.hide-text {
  font: a, sans-serif;
  font-size: 0;
  line-height: 0;
  color: transparent;
  text-shadow: none;
  background-color: transparent;
  border: 0;
}

Reply:
done